cargo-dist/templates/installer/homebrew.rb.j2

```diff
@@ -0,0 +1,16 @@
+class {{ formula_class }} < Formula
+  {%- if desc %} desc "{{ desc }}" {% endif %}
+  {%- if homepage %} homepage "{{ homepage }}" {% endif %}
+  if Hardware::CPU.type == :arm
+    url "{{ inner.base_url }}/{{ arm64.id }}"
+    sha256 "{{ arm64.checksum }}"
```

Comment:
Looks like these aren't generated right now; I'll have to figure out how to fix that.

Reply:
We do generate sha256 checksums for archives (as files), what do you need the shasum to cover? just the binary?

Reply:
It's for the tarball itself, rather than any of the contents.